samples/apps/copilot-chat-app/webapp/README.md

```diff
@@ -5,6 +5,73 @@ production deployments.
 
 See [../README.md](../README.md) for complete instructions on setting up and running the application.
 
+## How to use HTTPS for local development
+If you want to run Copilot Chat with HTTPS, you need to create a certificate and sign it with a Certificate Authority (CA) that is trusted locally by your device and browser. You can do that easily with [mkcert](https://github.com/FiloSottile/mkcert#installation).
```

Comment:
I like the mkcert path you took here. Could we also add a note in this first part that includes a link for the Azure Key Vault certificate creation process? https://learn.microsoft.com/en-us/azure/key-vault/certificates/certificate-scenarios. I don't think we need much more than just a link to their page for the cert creation. They also have a section on creating an official globally accepted SSL certificate.